pkg/syncer/syncer.go

```diff
@@ -67,13 +75,34 @@ func StartSyncer(upstream, downstream *rest.Config, resources sets.String, clust
 		specSyncer.Stop()
 		return nil, err
 	}
+	leaseClient, err := clientset.NewForConfig(upstream)
+	if err != nil {
+		specSyncer.Stop()
```

Comment:
Should this also stop the `statusSyncer`?

Reply:
✅